e2e/test/scenarios/question/notebook-native-preview-sidebar.cy.spec.ts

```diff
@@ -156,30 +158,35 @@ describe("scenarios > question > notebook > native query preview sidebar", () =>
       borderWidth -
       sidebarMargin;
 
+    cy.intercept("POST", "/api/dataset/query_metadata").as("metadata");
     cy.intercept("POST", "/api/dataset/native").as("nativeDataset");
+    cy.intercept("GET", "/api/session/properties").as("sessionProperties");
     cy.intercept("PUT", "/api/setting/notebook-native-preview-shown").as(
       "updatePreviewState",
     );
 
     openReviewsTable({ mode: "notebook", limit: 1 });
+    cy.wait("@metadata");
     cy.findByLabelText("View the SQL").click();
-    cy.wait("@updatePreviewState");
-    cy.wait("@nativeDataset");
+    cy.wait(["@updatePreviewState", "@sessionProperties"]);
 
     cy.log(
       "It should not be possible to shrink the sidebar below its min (initial) width",
     );
     resizeSidebar(200, (initialSidebarWidth, sidebarWidth) => {
-      expect(initialSidebarWidth).to.eq(minSidebarWidth);
-      expect(sidebarWidth).to.eq(initialSidebarWidth);
+      expect(initialSidebarWidth).to.be.closeTo(
+        minSidebarWidth,
+        toleranceDelta,
+      );
+      expect(sidebarWidth).to.be.closeTo(initialSidebarWidth, toleranceDelta);
```

Comment:
One other thing that @iethree and I observed last week was that it would fail because 427 was not equal to 427.20something.

This assertion adds some flexibility that doesn't diminish our confidence that this feature is working as intended.